features/step_definitions/web_steps.rb

```diff
@@ -0,0 +1,55 @@
+Given(/^I visit the homepage$/) do
+  visit('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.